libs/sdk-ui-kit/src/Header/HeaderInviteButton.tsx

```diff
@@ -0,0 +1,16 @@
+// (C) 2021-2022 GoodData Corporation
```

Comment:
As this is a new file, the copyright should be `2022` only, shouldn't it?

Reply:
I updated it thanks